lib/kafka_ex.ex

```diff
@@ -547,25 +547,9 @@ defmodule KafkaEx do
     default_api_versions = %{fetch: 0, offset_fetch: 0, offset_commit: 0}
     api_versions = Keyword.get(opts, :api_versions, %{})
     api_versions = Map.merge(default_api_versions, api_versions)
+    offset_fetch_api_version = Map.fetch!(api_versions, :offset_fetch)
 
-    retrieved_offset =
-      if consumer_group && !supplied_offset do
-        request = %OffsetFetchRequest{
-          topic: topic,
-          partition: partition,
-          consumer_group: consumer_group,
-          api_version: Map.fetch!(api_versions, :offset_fetch)
-        }
-
-        fetched_offset =
-          worker_name
-          |> KafkaEx.offset_fetch(request)
-          |> KafkaEx.Protocol.OffsetFetch.Response.last_offset()
-
-        fetched_offset + 1
-      else
-        current_offset(supplied_offset, partition, topic, worker_name)
-      end
+    current_offset(supplied_offset, partition, topic, worker_name, offset_fetch_api_version)
```

Comment:
The code that this replaces used `consumer_group` if it was passed. Shouldn't this functionality be preserved ?

Reply:
I was thinking of it, but if you look at the code, we commit 2 times, one for the consumer_group that is passed and also by the default_consumer group thought fetch with `auto_commit` set to true.

Therefore, you can end up by fetching one time with one consumer_group and committing the fetch, then all other fetch with the default worker will consume from the latest committed offset, meaning the first fetch of the instantiation of the stream. 

I may be wrong, but it seems to me that if we want to preserve this, we should not commit the offset when using the first fetch, and commit manually to the passed consumer group, or to never care of the consumer when using the first fetch

Reply:
Ok, I get it now.

In the `Server.call(%{:fetch})` of the stream, we do set the auto_commit to false. Therefore, we do not commit for the `fetch` but when the fetch has been perfomed by the `Stream`.

I'll add the `consumer_group` in the `current_offset()` function to be sure that we do fetch the last offset for the consumer_group passed to the Stream. Thanks for noticing it.